### PR TITLE
Add config service and new currencies

### PR DESCRIPTION
**Pull request by luciorubeens**
Used data from fixer.io to add BRL and KRW currencies. A future implementation to add/remove any currency as user prefer.

#110

### CODE REVIEW

File: client/app/src/accounts/AccountController.js [hunk 1]
```diff
@@ -71,7 +71,7 @@
    * @param avatarsService
    * @constructor
    */
-  function AccountController( accountService, networkService, storageService, changerService, $mdToast, $mdSidenav, $mdBottomSheet, $timeout, $interval, $log, $mdDialog, $scope, $mdMedia, gettextCatalog) {
+  function AccountController(configService, accountService, networkService, storageService, changerService, $mdToast, $mdSidenav, $mdBottomSheet, $timeout, $interval, $log, $mdDialog, $scope, $mdMedia, gettextCatalog) {
```
Comment (fix) [inline client/app/src/accounts/AccountController.js]: this is service is not used in AccountController

**Comment by fix** [comment]
I don't understand fully configService. I am not confortable with the concept: Should we migrate all "static" data inside? such as 
- https://github.com/ArkEcosystem/ark-desktop/blob/master/client/app/src/accounts/AccountController.js#L84
- https://github.com/ArkEcosystem/ark-desktop/blob/master/client/app/src/accounts/AccountController.js#L84

**Comment by fix** [comment]
Hi Lucio, anything on it. There is another PR #129 that could be linked to this. Are you working on a new version?

**Comment by luciorubeens** [comment]
Hi. Yes, i'm adding #124 too.

**Pull request closed without merging**